xarray/core/common.py

```diff
@@ -812,8 +812,8 @@ def rolling(
         center : bool or mapping, default: False
             Set the labels at the center of the window.
         keep_attrs : bool, optional
-            If True, the object's attributes (`attrs`) will be copied from
-            the original object to the new one.  If False (default), the new
+            If True (default), the object's attributes (`attrs`) will be copied
+            from the original object to the new one.  If False, the new
```

Comment:
```suggestion
            If None (default) or True, the object's attributes (`attrs`) will be
            copied from the original object to the new one. If False, the new
```

Reply:
I'd replace the `(default)` and the `, optional` in the type spec with `, default: None`